app/assets/javascripts/school_administrator_dashboard/dashboard_components/absences_dashboard/SchoolAbsenceDashboard.js

```diff
@@ -87,7 +88,7 @@ class SchoolAbsenceDashboard extends React.Component {
           seriesData = {filteredAttendanceSeries}
           yAxisMin = {80}
           yAxisMax = {100}
-          titleText = {'Average Attendance By Month'}
+          titleText = {'Average Attendance By Month (' + this.state.selectedRange + ')'}
```

Comment:
@edavidsonsawyer Could be a good spot for interpolation! 

```
`Average Attendance By Month (${this.state.selectedRange})`
```

Reply:
@alexsoble that's slick, let's do that